lib/mongo/collection.rb

```diff
@@ -228,6 +228,13 @@ def capped?
     #
     # @since 2.0.0
     def create(opts = {})
+      # Passing read options to create command causes it to break.
+      # Filter the read options out.
+      # TODO put the list of read options in a class-level constant when
```

Comment:
Do we have this list?

Reply:
No, this will need to be researched or done organically over time.